test/stdlib/TestJSONEncoder.swift

```diff
@@ -216,10 +216,21 @@ class TestJSONEncoder : TestJSONEncoderSuper {
   }
 
   // MARK: - Data Strategy Tests
+  func testEncodingData() {
+    let data = Data(bytes: [0xDE, 0xAD, 0xBE, 0xEF])
+
+    // We can't encode a top-level Data, so it'll be wrapped in a dictionary.
```

Comment:
These phrasing changes should have gone in in a previous PR but I missed them. May as well fix them here instead of propagating them.